spyder/widgets/tests/test_status.py

```diff
@@ -62,53 +66,75 @@ def test_status_bar_widget_signal(status_bar, qtbot):
     assert w.get_icon() == 'icon'
 
 
-def test_status_bar_conda_status(status_bar, qtbot, mocker):
-    mocker.patch.object(utils, 'is_anaconda', return_value=True)
+def test_status_bar_conda_interpreter_status(status_bar, qtbot, mocker):
+    # We patch where the method is used not where it is imported from
+    mocker.patch.object(spyder.widgets.status, 'is_conda_env',
```

Comment:
`is_conda_env` is a function of `spyder.utils.conda`. If possible, I think it's better to import it from that module directly instead of patching it indirectly through the import present in `spyder.widgets.status`.

Reply:
Hmm..., @goanpeca advised the current implementation because we couldn't get mocker to patch it when we imported from `spyder.utils.conda`, hence the comment `# We patch where the method is used not where it is imported from`.  
See resolved conversation above: https://github.com/spyder-ide/spyder/pull/12232#discussion_r421917790

If we missed something there, please let me know.

Reply:
Sorry, I missed that comment. It's a minor, nitpicky detail. The most important thing is to test this appropriately, which you did.